client/src/components/RhythmCodeIcon.js

```diff
@@ -40,14 +40,14 @@ function RhythmCodeIcon({
 			<div
 				className="rhythmItemList"
 				style={{
-					display: isDropdownShown ? "block" : "none",
+					display: isOn ? "block" : "none",
```

Comment:
Can we continue to call this `isDropdownShown` rather than `isOn`?

Reply:
name fixed